### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by Novfensec**
- This pull-request contains a recipe for [bitarray](https://github.com/ilanschnell/bitarray).
- Also contains bug fixes for `itsdangerous` recipe and `pillow` recipe by [@hemna](https://github.com/hemna) in extension to the pull-request at https://github.com/kivy/kivy-ios/pull/946 .
- Tested the recipes and built successfully an ipa for iOS.
- You can close https://github.com/kivy/kivy-ios/pull/946 with unmerged commits.

To support the latest pillow version, need to install these dependencies:
```bash
brew update
brew install autoconf automake libtool pkg-config libjpeg libtiff webp little-cms2
```

**Updated README.md with latest command.**

### CODE REVIEW

**Comment by Novfensec** [comment]
Please wait till Its ready for review.

**Comment by Novfensec** [comment]
@AndreMiras 
Updated README.md with latest commands.
The pull-request can be merged.

File: kivy_ios/recipes/pillow/__init__.py [hunk 1]
```diff
@@ -5,8 +5,9 @@
 
 
 class PillowRecipe(CythonRecipe):
-    version = "8.2.0"
-    url = "https://pypi.python.org/packages/source/P/Pillow/Pillow-{version}.tar.gz"
+    version = "11.0.0"
+    #url = "https://pypi.python.org/packages/source/p/pillow/pillow-{version}.tar.gz"
```
Comment (AndreMiras) [inline kivy_ios/recipes/pillow/__init__.py:9]: We can probably remove this commented out line

**Review by AndreMiras — DISMISSED** [review]
LGTM thanks.
Did you get a chance to test it runtime?
Let's see what the CI says at compile time.
Also it's not clear if the bitarray recipe is just an extra recipe you thought was nice to have or if it's actually used by pillow?

**Comment by Novfensec** [comment]
> LGTM thanks.
> Did you get a chance to test it runtime?
> Let's see what the CI says at compile time.
> Also it's not clear if the bitarray recipe is just an extra recipe you thought was nice to have or if it's actually used by pillow?

It adds to the existing recipes. I have used it in one of my projects and might someone else find errors packaging it to ios,so to save from errors.

**Comment by Novfensec** [comment]
removed commented line and added a line at then end of bitarray/__init__.py

**Pull request closed without merging**